services/eventsapi/main.py

```diff
@@ -69,4 +74,11 @@ async def create_event(event: Event):
                 Body=event.json()
             )
     else:
-        print(f"Received event: {event.json()}")
\ No newline at end of file
+        print(f"Received event: {event.json()}")
+
+    if KAFKA_BROKERS:
+        producer = KafkaProducer( bootstrap_servers=KAFKA_SERVER, api_version=(2, 0, 2))
```

Comment:
A couple of questions on this `api_version` argument:

1. I believe this is supposed to be the kafka broker API version
2. Do we need to pass it? It looks like if we don't it will try to probe the broker to negotiate version. Is that not working in this environment?